prow/config.yaml

```diff
@@ -5924,6 +5924,18 @@ periodics:
       args:
       - "--timeout=320"
       - "--bare"
+- name: ci-ingress-gce-e2e-scale
+  agent: kubernetes
+  interval: 360m
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180205-d99d9c246-master
+      args:
+      - "--timeout=320"
```

Comment:
hummm, the pod timeout need to be longer than config.json timeout, i.e. at least 340 here (our unit-test is indeed borked)

Reply:
Humm I had a wrong feeling that is no longer required, I was wrong :) Fixed.